README.rst

```diff
@@ -0,0 +1,61 @@
+sctools
+=======
+
+sctools provides utilities for manipulating sequence data formats suitable for use in distributed
+systems analyzing large biological datasets.
+
+Download and Installation
+-------------------------
+
+.. code bash
```

Comment:
Should this be uncommented or removed?

Reply:
That's how ReST-based markdown works -- it's not a comment; it's boilerplate to initiate a code block. We unfortunately need ReST for uploads to Pypi. :(

Reply:
Ohhh okay! Sorry, I just assumed it was a comment because the text is grey :/

Reply:
No need to apologize :)